HW02/Pavel_Karenda/pavel_karenda_hw_02_t_02.rb

```diff
@@ -0,0 +1,55 @@
+ERROR = /error/.freeze
+IP = /\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/.freeze
+DATE = %r{\d{1,2}/\w{3}/\d{4}\:\d{0,2}\:\d{0,2}\:\d{0,2}\s\+\d{4}}.freeze
+DESTINATION = %r{/\w+/\d+/\w+\s}.freeze
+arr_lines = []
+# 23/Apr/2018:20:30:39 +0300
+if ARGV.length != 1
+  puts 'We need exactly one parameter. The name of a file.'
+  exit
+end
+filename = ARGV[0]
+puts "Going to open '#{filename}'"
+
+file = File.open(filename)
```

Comment:
Choose the best name for the variable.
For example: _file_content_
Fix it in all tasks
```suggestion
file_content= File.open(filename)
```